docs/tutorials/AddRemark.md

```diff
@@ -292,7 +292,7 @@ While the changes to code may be minimal, the test data will have to be updated
 
 <div markdown="span" class="alert alert-warning">
 
-:exclamation: You must delete AddressBook’s storage file located at `/data/addressbook.json` before running it! Not doing so will cause AddressBook to default to an empty answer book!
+:exclamation: You must delete AddressBook’s storage file located at `/data/medmoriser.json` before running it! Not doing so will cause AddressBook to default to an empty answer book!
```

Comment:
'AddressBook' to 'Medmoriser'
'answer book' to 'question bank' or whatever term is appropriate?